prow/oss/cluster/crier.yaml

```diff
@@ -47,6 +50,9 @@ spec:
         - name: kubeconfig
           mountPath: /etc/kubeconfig
           readOnly: true
+        - name: build-kubeflow
```

Comment:
Would it be easier to use `secret.items` like https://github.com/openshift/release/blob/4b6b42276b95d8688c9327158b28aaa5dcdfd2c5/clusters/app.ci/prow/03_deployment/hook.yaml#L231?

Reply:
I don't see how that would help. All those keys are from the same secret which is not the case for us (and shouldn't be if we want to avoid secret size limits down the road). AFAICT that is just being used to rename the keys.

Reply:
Gotcha, overlooked the last line of the openshift config.